public/components/security/roles-mapping/components/roles-mapping-table.tsx

```diff
@@ -58,10 +60,23 @@ export const RolesMappingTable = ({ rolesEquivalences, rules, loading, editRule,
     {
       field: 'id',
       name: 'Status',
-      render: item => {
-        return WzAPIUtils.isReservedID(item) && <EuiBadge color="primary">Reserved</EuiBadge>;
+      render (item, obj){
+        if(WzAPIUtils.isReservedID(item)){
+          if( (obj.id == 1 || obj.id == 2)){
```

Comment:
it is recommended to use `===`

Reply:
Okay, it is changed and I will keep it in mind for future occasions. Thank you.